docker/python3/Dockerfile

```diff
@@ -14,3 +14,16 @@ RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-
 
 RUN addgroup -g 4000 demisto \
   && adduser -u 4000 -G demisto -D demisto -s /bin/sh 
+
+# Handling the issue described here https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000POJ0CAO&lang=en_US%E2%80%A9
+# by enabling UnsafeLegacyRenegotiation
+RUN echo 'openssl_conf = openssl_init\
```

Comment:
can we have this in the folder as a file and COPY it in?

Reply:
+1, there are other docker-images which should be updated, better to do in an external file and copy it